<?xml version="1.0"?>
<launch>
	<!--lcl-->
	<param name="lcl/save_num" type="int" value="50" />
    <param name="lcl/skip_cpunt" type="int" value="10" />
	<node pkg="sensor_fusion" type="lcl" name="lcl" >
		<remap from="/cloud" to="/cloud/tf" />
	</node>
</launch>
